src/components/Suggestor.jsx

```diff
@@ -37,6 +37,8 @@ const Suggestor = (props: Props) => {
   } = props;
   const [selectedMovie, setSelectedMovie] = useState('none');
   const [selectedYear, setSelectedYear] = useState('none');
+  const [selectedLink, setSelectedLink] = useState('none');
```

Comment:
Initialize it to null i.e useState(null) when value won't be used initially i.e here we don't have a link named 'none'.